cmd/validator/validator.go

```diff
@@ -10,9 +10,11 @@ Usage:
 
 The flags are:
     -exclude-dirs string
-    	Subdirectories to exclude when searching for configuration files
+    	Subdirectories to exclude when searching for configuration files.
 	-reporter string
 		Format of printed report. Currently supports standard, JSON.
+	-exclude-file-types string
```

Comment:
Looks like these comments got misaligned at some point with a tab starting at line 14. Please correct to replace tabs with spaces and align the options to line 12.